src/validationinterface.cpp

```diff
@@ -177,18 +177,18 @@ void SyncWithValidationInterfaceQueue()
 // evaluating arguments when logging is not enabled.
 //
 // NOTE: The lambda captures all local variables by value.
-#define ENQUEUE_AND_LOG_EVENT(event, fmt, name, ...)           \
-    do {                                                       \
-        auto local_name = (name);                              \
-        LOG_EVENT("Enqueuing " fmt, local_name, __VA_ARGS__);  \
-        m_internals->m_schedulerClient.AddToProcessQueue([=] { \
-            LOG_EVENT(fmt, local_name, __VA_ARGS__);           \
-            event();                                           \
-        });                                                    \
+#define ENQUEUE_AND_LOG_EVENT(category, event, fmt, name, ...)          \
```

Comment:
Note: I don't know if there was any particular reason why the endline characters are vertically aligned. If there's no reason, then it would be easier to move them all to the left.

cc @kczyz